legend-engine-xt-relationalStore-grammar/src/main/java/org/finos/legend/engine/language/pure/compiler/toPureGraph/HelperRelationalBuilder.java

```diff
@@ -961,6 +966,20 @@ private static PropertyMapping processRelationalPropertyMapping(RelationalProper
             Assert.assertTrue(eMap != null, () -> "Can't find enumeration mapping '" + propertyMapping.enumMappingId + "'");
             res = res._transformer(eMap);
         }
+        try
+        {
+            Pair<Type, Type> incompatiblePropertyMappingTypes = (Pair<Type, Type>) Root_meta_relational_mapping_getIncompatibleTypesInRelationalPropertyMapping_RelationalPropertyMapping_$0_1$__Pair_$0_1$_(res, context.pureModel.getExecutionSupport());
+
+            if (incompatiblePropertyMappingTypes != null)
+            {
+                context.pureModel.addWarnings(Lists.mutable.with(new Warning(SourceInformationHelper.fromM3SourceInformation(res.getSourceInformation()), "Type of class property \"" + res._property()._name() + "\" i.e " + incompatiblePropertyMappingTypes._first()._name() + " is incompatible with it's mapped relational property type i.e " + incompatiblePropertyMappingTypes._second()._name())));
+            }
+        }
+        catch (Exception exception)
+        {
+            LOGGER.warn("Ignored exception: ", exception.getMessage());
```

Comment:
LOGGER.warn("Ignored exception", exception); is better as it will log the whole trace